vllm/platforms/__init__.py

```diff
@@ -142,6 +142,7 @@ def xpu_platform_plugin() -> Optional[str]:
             logger.debug("Confirmed %s backend is available.",
                          XPUPlatform.dist_backend)
             logger.debug("Confirmed XPU platform is available.")
+            XPUPlatform.forward_api()
```

Comment:
Can we do that in xpu.py?

Reply:
in xpu.py, XPUPlatform doesn't have a init function. but we can port what in `forward_api` method into XPUPlatform attribute init part. we need make sure that being called in all process.

Reply:
yes, add __post_init__ in xpu.py maybe?